tests/ImageSharp.Tests/Drawing/SolidPolygonTests.cs

```diff
@@ -31,7 +31,7 @@ public void ImageShouldBeOverlayedByFilledPolygon()
             {
                 image
                     .BackgroundColor(Color.Blue)
-                    .FillPolygon(Color.HotPink, simplePath)
+                    .FillPolygon(Color.HotPink, simplePath, new GraphicsOptions(true))
```

Comment:
I'm assuming this is just to explicitly test the overload. Default is anti-alias yeah?